cli/BUILD.bazel

```diff
@@ -1,8 +1,13 @@
 load(
     "@gz//bazel/skylark:build_defs.bzl",
+    "GZ_ROOT",
     "GZ_VISIBILITY",
 )
 
+package(
+    default_applicable_licenses = [GZ_ROOT + "utils:license"],
```

Comment:
Should this be pointing to the license of `cli`?

Reply:
Correct, updated